returnn/tf/layers/basic.py

```diff
@@ -4252,7 +4252,7 @@ def __init__(self, filter_size, activation, strides=None,
     padding = padding.lower()
     if strides is None:
       strides = filter_size
-    if not isinstance(remove_padding, (tuple, int)):
+    if not isinstance(remove_padding, (tuple, tuple)):
```

Comment:
I assume this was supposed to mean:
```suggestion
    if not isinstance(remove_padding, (list, tuple)):
```

Reply:
Oh yes of course, sorry...